app/src/routes/authorizedRoute.jsx

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { redirect as rfrRedirect } from 'redux-first-router';
-import { isAuthorizedSelector } from 'controllers/auth';
+import {
+  isAuthorizedSelector,
+  isAdminAccessSelector,
+  resetAdminAccessAction,
+} from 'controllers/auth';
 import { LOGIN_PAGE } from 'controllers/pages';
 
 export const authorizedRoute = (Component) =>
   connect(
     (state) => ({
       authorized: isAuthorizedSelector(state),
+      isAdminAccess: isAdminAccessSelector(state),
     }),
     (dispatch) => ({
       redirect: () => dispatch(rfrRedirect({ type: LOGIN_PAGE })),
+      resetAdminAccess: () => dispatch(resetAdminAccessAction()),
```

Comment:
resetAdminAccess: resetAdminAccessAction

Reply:
With this changes action will never dispatched